doc/output-details-and-examples/src/output-files/eplusout-eio.tex

```diff
@@ -2366,7 +2345,15 @@ \subsubsection{Field: Roughness}\label{field-roughness-1}
 
 \subsubsection{Field: Conductance}\label{field-conductance}
 
-This is the center-of-glass conductance of the construction calculated with film coefficients, for winter conditions (indoor air temperature = 21C (69.8F), outdoor air temperature = -18C (-.4F), windspeed = 5.5 m/s (12.3 mph), no solar radiation). Units are W/m\(^{2}\)-K.
+This is the center-of-glass conductance of the construction calculated with film coefficients, for winter conditions (indoor air temperature = 21C (69.8F), outdoor air temperature = -18C (-.4F), windspeed = 5.5 m/s (12.3 mph), no solar radiation). Units are W/m\(^{2}\)-K. The value is calculated by assuming a simple glazing plane with film coefficients, while the film coefficients can be adjusted assuming highly conducted frames are applied given high Nominal U input.
+
+\subsubsection{Field: Conductance (Before Adjusted)}\label{field-conductance-before-adjusted}
+
+This is the center-of-glass conductance of the construction calculated with film coefficients, assuming glazing only, for winter conditions (same as above). Units are W/m\(^{2}\)-K. The value can be lower than the input Nominal U for the simple glazing system due to heat transfer physics limit assuming the default film coefficients and a flat plane.
+
+\subsubsection{Field: Convection Coefficient Adjustment Ratio}\label{field-convection-coefficient-adjustment-ratio}
+
+For simple glazing (WindowMaterial:SimpleGlazingSystem), this is the adjustment ratio to both inside and outside window film coefficients assuming highly conducting frames are applied to achieve a high Nominal U-Factor. The adjustment ratio is calculated such that the glazing-only conductance evaluated at winter conditions equals the input Nominal U-Factor.
```

Comment:
@xuanluo113 I did some more cleanup and updated the text for this field. Please confirm if this is accurate.

Reply:
@mjwitte Reads great.